package.json

```diff
@@ -188,15 +188,15 @@
     "@types/js-cookie": "3.0.2",
     "@types/node": "14.14.25",
     "@types/react": "17.0.44",
-    "@typescript-eslint/eslint-plugin": "5.30.6",
-    "@typescript-eslint/parser": "5.30.6",
-    "eslint": "8.17.0",
-    "eslint-config-prettier": "8.5.0",
-    "eslint-plugin-import": "2.26.0",
-    "eslint-plugin-prettier": "4.2.1",
-    "eslint-plugin-react": "7.29.4",
-    "eslint-plugin-react-hooks": "4.6.0",
-    "eslint-plugin-unused-imports": "2.0.0",
+    "@typescript-eslint/eslint-plugin": "8.1.0",
```

Comment:
Did you have to upgrade those to support @symmetry-hq/basket-sdk?

Reply:
?? https://github.com/solana-labs/governance-ui/blob/23233f1d86b6897c31dbf204af80c93980e1c9e9/package.json#L190

this is the package.json of current main branch of solana-labs/governance-ui.

I didn't upgrade anything